scripts/setup

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3 -u
+from rever.main import setup
+setup()
```

Comment:
I don't think there should be a top-level setup script. Rather the normal rever command should have a setup argument

Reply:
How do we trigger the setup?

Reply:
Should there be a -s flag?

Reply:
Yeah, -s or --setup